pyslackersweb/website/tasks.py

```diff
@@ -1,17 +1,27 @@
 import dataclasses
+import json
 import logging
 from collections import Counter
-from typing import List, Awaitable, Callable
+from typing import List
 
 import slack
+from aiohttp import ClientSession
+from aioredis.abc import AbcConnection
+from slack.io.abc import SlackAPI
 
-from aiohttp import web
-
-from .util.log import ContextAwareLoggerAdapter
+from pyslackersweb.util.log import ContextAwareLoggerAdapter
 
 
 logger = ContextAwareLoggerAdapter(logging.getLogger(__name__))
 
+GITHUB_REPO_CACHE_KEY = "github:repos"
+
+SLACK_CHANNEL_CACHE_KEY = "slack:channels"
+
+SLACK_COUNT_CACHE_KEY = "slack:user:count"
+
+SLACK_TZ_CACHE_KEY = "slack:user:timezones"
```

Comment:
We should have key either singular or plural IMO
`slack:users` & `slack:channels`
`slack:user` & :slack:channel`